src/Views/home.blade.php

```diff
@@ -141,16 +141,18 @@
 
 	            <div class="col-md-4">
 		            <!-- CATEGORY -->
-			            <select id="chatter_category_id" class="form-control" name="chatter_category_id">
-			            	<option value="">@lang('chatter::messages.editor.select')</option>
-				            @foreach($categories as $category)
-				            	@if(old('chatter_category_id') == $category->id)
-				            		<option value="{{ $category->id }}" selected>{{ $category->name }}</option>
-				            	@else
-				            		<option value="{{ $category->id }}">{{ $category->name }}</option>
-				            	@endif
-				            @endforeach
-			            </select>
+					<select id="chatter_category_id" class="form-control" name="chatter_category_id">
+						<option value="">@lang('chatter::messages.editor.select')</option>
+						@foreach($categories as $category)
+							@if(old('chatter_category_id') == $category->id)
+								<option value="{{ $category->id }}" selected>{{ $category->name }}</option>
+							@elseif(!empty($category_id) && $category_id == $category->id)
```

Comment:
I had trouble with the mix of tabs and spaces in this file. So although it looks like it all changed, I only added the middle `@elseif` part.

It could technically be merged with the first `if`, since they both just add `selected` but I feel this is more readable. I.e. You either get selected because you match the old input data or you get selected because you're on a category page at the moment.